hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/commit/UpsertPartitioner.java

```diff
@@ -379,6 +374,9 @@ protected static long averageBytesPerRecord(HoodieTimeline commitTimeline, Hoodi
         Iterator<HoodieInstant> instants = commitTimeline.getReverseOrderedInstants().iterator();
         while (instants.hasNext()) {
           HoodieInstant instant = instants.next();
+          if (instant.getAction().equals(REPLACE_COMMIT_ACTION)) {
+            continue;
```

Comment:
Would prefer `(CollectionUtils.createSet(COMMIT_ACTION, DELTA_COMMIT_ACTION)`, one thing needs to note is the avro file uses the Java reflection to calcute the in-memory size of a record, while Parquet file size is the actual file size so it is more accurate, not sure whether we should include the log if there are parquets, that means, the parquet file size should have higher priority than log.

Reply:
hi @danny0405, I just reverted the code to the very first one.  
Whether including the log or not, the current one would be better than the released one, at least for MoR tables

Reply:
Agree, it is great if you can add a test for for the method.